views/secretWord.ejs

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
You can remove this code because it is included in the head.ejs and headwe.ejs, which you are including bellow.